src/java/picard/sam/markduplicates/MarkDuplicates.java

```diff
@@ -64,28 +64,35 @@
-    static final String USAGE_DETAILS =
-            "This tool locates and tags duplicate reads (both PCR and optical/sequencing-driven) in a BAM or SAM file, where\n" +
-                    "duplicate reads are defined as originating from the same original fragment of DNA. Duplicates are identified as read\n" +
-                    "pairs having identical 5' positions (coordinate and strand) for both reads in a mate pair (and optionally, matching\n" +
-                    "unique molecular identifier reads; see BARCODE_TAG option). Optical, or more broadly Sequencing, duplicates are\n" +
-                    "duplicates that appear clustered together spatially during sequencing and can arise from optical/image-processing\n" +
-                    "artifacts or from bio-chemical processes during clonal amplification and sequencing; they are identified using the\n" +
-                    "READ_NAME_REGEX and the OPTICAL_DUPLICATE_PIXEL_DISTANCE options.\n" +
-                    "\n" +
-                    "The tool's main output is a new SAM or BAM file in which duplicates have been identified in the SAM flags field, or\n" +
-                    "optionally removed (see REMOVE_DUPLICATE and REMOVE_SEQUENCING_DUPLICATES), and optionally marked with a duplicate type\n" +
-                    "in the 'DT' optional attribute. In addition, it also outputs a metrics file containing the numbers of\n" +
-                    "READ_PAIRS_EXAMINED, UNMAPPED_READS, UNPAIRED_READS, UNPAIRED_READ DUPLICATES, READ_PAIR_DUPLICATES, and\n" +
-                    "READ_PAIR_OPTICAL_DUPLICATES.\n" +
-                    "\n" +
-                    "Usage example: java -jar picard.jar MarkDuplicates I=input.bam \\\n" +
-                    "                 O=marked_duplicates.bam M=marked_dup_metrics.txt\n" +
-                    "\n" +
-                    "The program can take either coordinate-sorted or query-sorted input, however the behavior is slightly different.\n" +
-                    "When the input is coordinate-sorted, unmapped mates of mapped records, and supplementary and secondary alignments are not\n" +
-                    "marked as duplicates. When the input is query-sorted (actually query-grouped) then unmapped mates get marked as their mapped\n" +
-                    "counter-parts, and secondary and supplementary reads get marked as the primary records with the same query-name.\n";
+    static final String USAGE_DETAILS = "<p>This tool locates and tags duplicate reads in a BAM or SAM file, where duplicate reads are " +
+            "defined as originating from a single fragment of DNA.  Duplicates can arise during sample preparation " +
+            "e.g. library construction using PCR.  For additional information on PCR-associated duplication artifacts, please see the documentation for the <a href='https://broadinstitute.github.io/picard/command-line-overview.html#EstimateLibraryComplexity'>EstimateLibraryComplexity</a> tool.  " +
```

Comment:
"For additional information on PCR-associated duplication artifacts" makes me think I'm going to find more detailed documentation on duplication mechanisms etc, but the link points to a tool doc that doesn't say much more about that beyond a short summary. The ideal solution would be to make a dictionary entry about duplicates and point to that -- but that's not a priority right now, so instead, just scale down the reader's expectations by saying eg "(see also EstimateLibraryComplexity for additional notes on PCR duplication)", with the link attached to EstimateLibraryComplexity.